digdag-core/src/main/java/io/digdag/core/database/DatabaseSessionStoreManager.java

```diff
@@ -149,7 +151,9 @@ public DatabaseSessionStoreManager(DBI dbi, ConfigFactory cf, ConfigMapper cfm,
         dbi.registerMapper(new InstantMapper());
         dbi.registerArgumentFactory(cfm.getArgumentFactory());
 
-        this.mapper = mapper;
+        this.taskArchiveMapper = mapper.copy();
+        this.taskArchiveMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
@komamitsu In fact, this line is not the main fix. This line is for this: https://github.com/treasure-data/digdag/pull/467/files#diff-b8b63c99b1a34f8b65c161e6885fa07cR477
The main fix (fix for adding columns) is this: https://github.com/treasure-data/digdag/pull/467/files#diff-a6a683385d89bdbb7743550326a8645eR38